.travis.yml

```diff
@@ -2,13 +2,48 @@ language: android
 
 android:
   components:
-    - platform-tools
+    # Update tools and then platform-tools explicitly so lint gets an updated database. Can be removed once 3.0 is out.
     - tools
-    - build-tools-23.0.1
-    - android-23
-    - extra-google-m2repository
-    - extra-android-m2repository
+    - platform-tools
 
 jdk:
   - oraclejdk8
 
+before_install:
+  # Install SDK license so Android Gradle plugin can install deps.
+  - mkdir "$ANDROID_HOME/licenses" || true
+  - echo "$LICENSES_HASH" > "$ANDROID_HOME/licenses/android-sdk-license"
```

Comment:
You can pull these from the AutoDispose repo settings, will need to set them up in your travis CI envs

Reply:
I already did 👀